compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Accessibility.uikit.kt

```diff
@@ -1110,12 +1124,20 @@ internal class AccessibilityMediator(
             "Root view must not be an accessibility element"
         }
 
-        root.element = traverseSemanticsTree(rootSemanticsNode)
+        val (element, focusedElementKey) = traverseSemanticsTree(rootSemanticsNode)
+        root.element = element
 
         accessibilityDebugLogger?.let {
             debugTraverse(it, view)
         }
 
+        if (forceFocusedElementKey != focusedElementKey) {
+            forceFocusedElementKey = focusedElementKey
+            focusedElementKey?.let {
```

Comment:
if there is no focused elemet (`focusedElementKey == null`) don't we want to set the `focuseMode` to `None` here?